packages/react-components/src/components/abstract-xy-output-component.tsx

```diff
@@ -269,6 +269,19 @@ export abstract class AbstractXYOutputComponent<P extends AbstractOutputProps, S
                     outputStatus: treeResponse.status,
                     xyTree: treeResponse.model.entries,
                     columns
+                }, () => {
+                    // Only call updateXY here when the chart has a default series to plot.
+                    // If it prompts the user to "select a checkbox to see the analysis results",
+                    // updateXY will be called later.
+                    switch (this.props.outputDescriptor.name) {
+                        case 'CPU Usage':
```

Comment:
This is back-end dependent and can't be merged like that. I don't understand why the xy charts should only be updated for certain xy charts and not for others. Please note that some back-ends don't have CPU usage nor Memory Usage views. When I ran it, the histogram was not updated. There was one rendering of XY data, but not more with the large trace I used to test. 

The goal of the PR is to show XY data in the overview view while analysis is still running, independent what type of XY chart the user selected. Scope of the implementation beside functionality is also performance and UX. If the performance is not acceptable, then that needs to be addressed as well.

Please test with large traces where the analysis takes a long time (e.g. maybe > 1 minute).

Reply:
Check for 'CPU Usage' removed.